### PR TITLE
C1 nav bar/home page

### PR DESCRIPTION
**Pull request by devmarantz**
All my hard work

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,17 @@
+{
```
Comment (samsaksfithian) [inline .vscode/settings.json:1]: Pls add `.vscode` to your `.gitignore`

File: src/App.js [hunk 2]
```diff
@@ -1,9 +1,24 @@
 import React from 'react';
-import './css/main.css';
-import { LessonInfo } from './components';
+// import PropTypes from 'prop-types';
+import './css/main.scss';
```
Comment (samsaksfithian) [inline src/App.js:3]: I think CSS imports should be below your other imports, but it doesn't matter that much.

File: src/components/pages/HomePage.js [hunk 3]
```diff
@@ -1,5 +1,116 @@
...
+              </div>
+            </section>
+            <section className="company-box">
+              <img src="https://img.icons8.com/ios-filled/50/000000/investment.png" />
+              <div className="company-info">
+                <p>Get paid to learn and launch your career? We know it might sound too good to be true. We can afford to make an upfront investment in your career because clients help us recoup our investment during your tenure as a Talent Path consultant</p>
+              </div>
+            </section>
+            <section className="company-box">
+              <img src="https://img.icons8.com/ios-filled/50/000000/knowledge-sharing.png" />
```
Comment (samsaksfithian) [inline src/components/pages/HomePage.js]: We probably should store this image locally so it's bundled with the site and we don't need to externally fetch it

File: src/css/main.scss [hunk 4]
```diff
@@ -10,8 +10,8 @@
 /* =========================== MAIN =========================== */
 
 body {
-  margin-top: var(--nav-height);
-  background-color: var(--wooloo);
+  /* margin-top: var(--nav-height); */
+  background-color: white;
```
Comment (samsaksfithian) [inline src/css/main.scss]: Gross, pure white

File: src/css/navbar.scss [hunk 5]
```diff
@@ -0,0 +1,36 @@
+#nav-main {
```
Comment (samsaksfithian) [inline src/css/navbar.scss:1]: Pls rename this file to `NavBar.scss`

File: src/components/NavBar.js [hunk 6]
```diff
@@ -1,5 +1,25 @@
 import React from 'react';
+import { NavLink } from 'react-router-dom';
+import '../css/NavBar.scss';
 
-const NavBar = () => <div>I am a navigation bar!</div>;
+const links = {
+  home: '/',
+  paths: '/path',
+  catalog: '/catalog',
+  support: '/login',
```
Comment (samsaksfithian) [inline src/components/NavBar.js:11]:
```suggestion
  support: '/support',
  login: '/login',
```

**Review by samsaksfithian — CHANGES_REQUESTED** [review]
Mostly good, but just with a couple minor things that should be tweaked

File: src/components/pages/HomePage.js (hunk 3 above)
Comment (afterafx) [reply src/components/pages/HomePage.js]: Okay I'll work on that later.

**Pull request closed without merging**